windows/src/desktop/kmshell/util/Keyman.System.KeymanStartTask.pas

```diff
@@ -53,68 +57,115 @@ class procedure TKeymanStartTask.CreateTask;
   pEventTrigger: IEventTrigger;
   pAction: IExecAction;
 begin
-  pService := CoTaskScheduler_.Create;
-  pService.Connect(EmptyParam, EmptyParam, EmptyParam, EmptyParam);
-
-  // Create or open the Keyman task folder
   try
-    pTaskFolder := pService.GetFolder('\' + CTaskFolderName);
+    pService := CoTaskScheduler_.Create;
+    pService.Connect(EmptyParam, EmptyParam, EmptyParam, EmptyParam);
+
+    // Create or open the Keyman task folder
+    try
+      pTaskFolder := pService.GetFolder('\*' + CTaskFolderName);
```

Comment:
What does `\*` do?

Reply:
Causes an error. Oops. That was debug code.